lib/flat_search_web/live/page_live.ex

```diff
@@ -18,10 +24,16 @@ defmodule FlatSearchWeb.PageLive do
       |> Filters.change_filter(params)
       |> Map.put(:action, :insert)
 
-    params = Enum.map(params, fn {key, value} -> {key, insensitive_string(value)} end)
-
     {:noreply,
-     assign(socket, changeset: changeset, flats: Flats.get_flats_by(params), filters: params)}
+     assign(socket,
+       changeset: changeset,
+       flats:
+         Flats.get_flats_by(
+           Enum.map(params, fn {key, value} -> {key, insensitive_string(value)} end),
+           params["order_by"]
+         ),
```

Comment:
maybe move this above the tuple and assign to variable `flats`

Reply:
I can't  override the original param because I still use it

Reply:
I did it this way not to allocate memory for only one use